migrations/00127_users_users.sql

```diff
@@ -0,0 +1,71 @@
+-- +goose Up
+/***********************************************************/
+/*** SCRIPT AUTHOR: Fredrik Vedvik (fredrik@vedvik.tech) ***/
+/***    CREATED ON: 2023-02-07T08:35:33.239Z             ***/
+/***********************************************************/
+
+--- BEGIN CREATE TABLE "users"."users" ---
+
+CREATE TABLE IF NOT EXISTS "users"."users" (
+	"id" varchar NOT NULL  ,
+	"email" varchar NOT NULL  ,
+	"display_name" varchar NULL  ,
+	"age" int4 NULL  ,
+	"church_ids" _int4 NULL  ,
+	"active_bcc" bool NULL  ,
```

Comment:
Any special reason for this to be nullable?

Reply:
Was for debug reasons, but no. Will fix the small issues